packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -129,7 +129,7 @@ const PlanFeatures2023GridActions = ( {
 			planSlug={ planSlug }
 			onClick={ callback }
 			busy={ busy }
-			disabled={ status !== 'enabled' }
+			disabled={ ! callback || 'disabled' === status }
```

Comment:
Sets the disabled to the same value as done here:

https://github.com/Automattic/wp-calypso/blob/a5a6b0f04e5565a1fbb0d093e46a1366f3a514c2/packages/plans-grid-next/src/components/actions.tsx#L183